project/BuildSettings.scala

```diff
@@ -0,0 +1,64 @@
+import sbt._
+import sbt.Keys._
+import sbtassembly.Plugin._
+import AssemblyKeys._
+
+import sbtbuildinfo.Plugin._
+
+
+
+import com.socrata.cloudbeessbt.SocrataCloudbeesSbt.{socrataBuildSettings, socrataProjectSettings}
+
+
+object BuildSettings {
+  val Organization = "com.socrata"
+
+  val buildSettings = socrataBuildSettings ++ buildInfoSettings ++
+    Seq(
+      name := "geospace",
+      scalaVersion := "2.10.3",
+      organization := Organization,
+      autoAPIMappings := true,
+      apiMappings += (
```

Comment:
actually, it was my attempt to get
@param layer an [[org.geoscript.layer.Layer]] or GeoTools FeatureSource.

to work, as it will not compile, in a previous pull request, I tried to address this, and was given some suggestions on how about doing it. It hasn't worked, so this is still an open item to address.